### ISSUE TITLE
Global scan/update is not updating up-to date manga (no unread chapters) 

### ISSUE THREAD

**Issue by bailinhuang** [mihonapp/mihon#238]
### Steps to reproduce

This is happening with Asura Scans. 

1. Install Asura Scans.
2. Add "Mount Hua Sect" manhwa.
3. Read all chapters. 
4. Make global update.
6. Try manual update inside the manhwa page.

### Expected behavior

After step 4, it should show the updated new chapter. 

### Actual behavior

It isn't showing unless you manually update 

### Mihon version

0.16.1

### Android version

14

### Device

Google Pixel 7

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.16.1](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by MajorTanya** [comment]
As said above, this sounds like intended/expected behaviour from the "Predict next chapter release" setting of the Smart Updates feature. Specifically, Mihon estimates the next chapter upload based on the recent few uploads.
With this, Mihon tries to reduce the number of unnecessary requests to sources, and thereby reduce the load on them.
Over time, Mihon should be getting a bit better at these estimations, but since they're linked to the upload dates, it can get thrown off if the schedule is particularly erratic.

As you mentioned yourself, you can always manually check a series if you e.g. know through other means that Mihon is wrong and there is a chapter now.

Generally, you won't "miss" any chapters, but they might appear a bit delayed for you. Overall, not a bug/issue that needs fixing, so I will close this issue.

**Comment by bailinhuang** [comment]
I understand. The only thing I am confused about is how it works, the following example:

If the release is every 4 days, why check every 12 instead of 4? 
![Uploading Screenshot_20240206-191805.png…]()

**Comment by MajorTanya** [comment]
Your screenshot didn't fully upload I think.
